getting_hired/preparing_for_job_search/project_portfolio.md

```diff
@@ -1,6 +1,6 @@
 ### Introduction
 
-You don't need a personal website right off the bat but it's good to have and it's a fun project to test your skills with.  In the simplest case, it's a direct conduit to you for anyone searching and another way to craft your narrative.  It doesn't have to be elaborate at all.  If you hate writing, don't blog.  But you need to be a real person.  Showcase your work.  Geek out.
+You don't need a personal website right off the bat but it's good to have and it's a fun project to test your skills with.  In the simplest case, it's a direct conduit to you for anyone searching and another way to craft your narrative.  It doesn't have to be elaborate at all.  If you hate writing, don't blog.  But you need to be a real person.  Showcase your work.Geek out.
```

Comment:
The removal of the double space eliminated all spaces, so this will need a space between the last two sentences.